### ISSUE TITLE
PvP Rework - GREF

### ISSUE THREAD

**Issue by ComboTombo** [ComboTombo/A3-Antistasi#4]
Primary Weapon will be set to AK-105 (Zenitco) with a PK-AS, IR/flashlight combo (Perst-3/2DP), and bog standard 5.45x39mm 7N6M ammunition.

_The AK-105's ammunition is compatible with the CDF infantry AK-74 (both use 5.45x39mm cartridges)_

Secondary weapon will be set to the PM Makarov.

_Outdated Soviet-bloc pistol. Makes sense for CDF to have it_

Uniforms will be set to the M88 TTsKO (Mountain). 

Vests will be set to the TTsKO (Mountain) 6B23.

_A good balance between the OP 6B13 and the "vests" that offer no protection._

Each vest will have 2 smoke nades, 1-2 fragmentation nades, 4-5 mags, and a pistol mag.

Helmets will be set to the Kaska K93 TTsKO (Mountain).

Team Leader gets a GPS and a PDU-4 Rangefinder.

_The reason why the team leader gets a rangefinder is that it incentivizes teamwork between the team leader and marksman, or team leader and AT._

NVGs will be set to the 1PN138.

_Closest thing to CDF would be AFRF so they have the same NVGs (and because GREF doesn't have an actual NVG in it)_

### SPECIAL FOR AT AND AAT ###

AT gets an RPG-7V2 with a PG-7VL rocket loaded and another rocket in a backpack.

_The AT gets the HEAT rockets, designed specifically for APCs & IFVs_

AAT gets an RPG-75 with two PG-7VR rockets in their backpacks.

_The AAT gets the RPG-7 Tandem HEAT rockets designed specifically for tanks and vehicles with ERA_

### SPECIAL FOR MACHINEGUNNER ###

MG gets an M249 PIP with an EOTech XPS3

_Lightweight machine gun used by CDF paratroopers. It would make sense for special forces to carry it. If the Taliban can have it, then they can have it too!_

MG vest gets two 100rnd boxes

_The already relatively heavy machinegunner will only get 300 rounds of ammunition_

### SPECIAL FOR MARKSMAN ###

Marksman gets an SVDS with a PSO-1M2 and 7N14 ammunition

_An earlier version of the SVD-series DMR. The loadout would've gotten the Dragunov if it existed in RHS_

Marksman vest has 9 10rnd 7N1 mags

_All marksmen loadouts will get exactly 100 rounds of ammunition. If you can't hit a target with 100 rounds, a high powered scope, and a marksman rifle, you should give the role to someone else that can_

**Comment by SonorousMeerkat** [comment]
It's a shame we don't have a "proper" rpd/rpk in RHS yet. The mags are there if you pick a suitable rifle though... (pm-63? or any other full length AK)

Only reason i mention is because they use 100% russian kit except that saw.

**Comment by ComboTombo** [comment]
Yeah, until RHS adds an RPK/RPD they'll have to stick with the SAW. But, it's within reason though. It isn't rare to find someone carrying a SAW in insurgent groups like Taliban, Al-Qaeda, and ISIS. If they can have it, then CDF can too.

**Comment by SonorousMeerkat** [comment]
CDF isn't an insurgent force though. It's the army of a minor eastern European country.

**Comment by ComboTombo** [comment]
I know, I'm just saying that if a militia and insurgency can get their hands on it, then an eastern european military sure can too. Plus, the SAW is available in the CDF paratrooper MG.
